src/utils/IDMassAccuracy.cpp

```diff
@@ -237,6 +237,9 @@ class TOPPIDMassAccuracy :
               md.exp_mz = it->getMZ();
               md.theo_mz = (hit.getSequence().getMonoWeight() + (double)charge * Constants::PROTON_MASS_U) / (double)charge;
               md.charge = charge;
+              // TODO
+              // md.intensity =  ??
+              // cppcheck-suppress uninitStructMember
               precursor_diffs.push_back(md);
```

Comment:
@timosachsenberg can you fix this?

Reply:
struct MassDifference
{
  double exp_mz = 0.0;
  Int charge  = 0;
  double theo_mz = 0.0;
  double intensity = 0.0;
};
should work